packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -466,6 +466,7 @@ export const style = createTheme({
         isFocusVisible: weirdColorToken('negative-background-color-key-focus'),
         isPressed: weirdColorToken('negative-background-color-down')
       },
+      // these are now deprecated (remove? doesn't seem like it's being used)
```

Comment:
yeah, go ahead and remove

Reply:
This is also in https://github.com/adobe/react-spectrum/pull/6940